src/components/badge/badge.scss

```diff
@@ -5,21 +5,28 @@
  * @prop --badge-text-color: badge text color
  */
 
-
 :host {
-  display: block;
+    display: block;
+}
+
+:host([hidden]) {
+    display: none;
+}
+
+* {
+    box-sizing: border-box;
 }
 
 .badge-container {
     display: inline-block;
-    font-size: 0.7rem;
-    min-width: 1.2rem;
-    height: 1.2rem;
-    border-radius: 1.2rem;
-    line-height: 1.2rem;
+    font-size: pxToRem(11);
+    min-width: pxToRem(20);
```

Comment:
Any advantages changing rem -> pxToRem(11)?

Reply:
The advantages, as I see them, is:

- (I think most of us tends to think in pixels, and if you do) `pxToRem(10)` is much easier to understand than `0.625rem`
- Since the vast majority of users have the default font size of 16px in their browser, using `pxToRem(x)`, where `x` is an integer, ensures that the end result is a rem value that translates to an integer number of pixels for that vast majority of users.

Nowadays, whole pixels are (at least almost always) rendered in a very predictable manner across different browsers and platforms. With most of our target browsers now using variants of Chromium, I imagine fractions of pixels are probably rendered a lot more similarly across browsers than only a few years ago, but I wouldn't trust the outcome on one machine to necessarily translate across the spectrum.

Reply:
Before:
![image](https://user-images.githubusercontent.com/154725/78232775-07722a80-74d5-11ea-8c5f-f8c1dc9478d0.png)
Only changing the box-sizing:
![image](https://user-images.githubusercontent.com/154725/78328990-ca637200-7580-11ea-8634-935b46ed323f.png)
Both changing the box-sizing and using whole pixels:
![image](https://user-images.githubusercontent.com/154725/78232795-0e993880-74d5-11ea-964b-01af74bf415d.png)

Notice especially that the red badge isn't round in the middle image.